models/engine/db_storage.py

```diff
@@ -74,3 +74,22 @@ def reload(self):
     def close(self):
         """call remove() method on the private session attribute"""
         self.__session.remove()
+
+    def get(self, cls, id):
+        """A method to retrieve one object:"""
+        if cls in classes.values():
+            a = self.all(cls)
```

Comment:
hey could u rename variable ```a``` to something relevant to what the function do.
naming examples: ```objects```,

Reply:
ok